ember-toucan-core/src/components/form/file-input-field.hbs

```diff
@@ -0,0 +1,112 @@
+<div
+  class="flex flex-col {{if @isDisabled 'text-disabled cursor-not-allowed'}}"
+  data-root-field={{if @rootTestSelector @rootTestSelector null}}
+>
+  <Form::Field as |field|>
+    {{!-- <field.Label for={{field.id}} data-label>{{@label}}</field.Label> --}}
+    <field.Label for={{field.id}} data-label>
+      {{@label}}
+      {{#if @hint}}
+        <field.Hint id={{field.hintId}} data-hint>{{@hint}}</field.Hint>
+      {{/if}}
+
+      <field.Control class="flex">
+        <div
+          class="bg-overlay-1 shadow-focusable-outline focus-within:shadow-focus-outline relative inline-flex h-8 w-full flex-wrap items-center rounded-sm"
+        >
+          <input
+            aria-describedby="{{if @error field.errorId}} {{if
+              @hint
+              field.hintId
+            }}"
+            {{! template lint thinks this input is a button, and so doesn't not support aria-invalid }}
+            {{! template-lint-disable no-unsupported-role-attributes }}
+            aria-invalid={{if @error "true"}}
+            class="text-titles-and-attributes focus:outline-none absolute m-1 h-6 w-0 cursor-pointer bg-transparent p-1
+              {{if @isDisabled 'text-disabled' 'text-titles-and-attributes'}}
+              {{if @error 'shadow-error-outline' 'shadow-focusable-outline'}}"
+            disabled={{@isDisabled}}
+            id={{field.id}}
+            type="file"
+            accept={{this.accept}}
+            multiple={{this.multiple}}
+            {{! template-lint-disable no-bare-strings no-inline-styles  }}
+            style="clip: rect(1px, 1px, 1px, 1px);"
+            ...attributes
+            {{on "change" this.handleChange}}
+          />
+          <div
+            class="text-titles-and-attributes inline-flex w-full cursor-pointer items-center pl-2"
+          >
+            {{! TODO: replace with official icon later }}
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              width="24"
+              height="24"
+              stroke="currentColor"
+              viewBox="0 0 24 24"
+            >
+              <path
+                d="M16.2 7.6l-2.1-2.3L12 3v13.1M12 3L9.9 5.3 7.8 7.6M21 15.1V19a2 2 0 01-2 2H5a2.006 2.006 0 01-2-2v-3.9"
+                fill="none"
+                stroke-linecap="round"
+                stroke-linejoin="round"
+                stroke-width="2"
+              ></path>
+            </svg>
+            {{#if (has-block "triggerText")}}
+              {{yield to="triggerText"}}
+            {{/if}}
+          </div>
+        </div>
+      </field.Control>
+    </field.Label>
+    {{#if @error}}
+      <field.Error id={{field.errorId}} data-error>{{@error}}</field.Error>
+    {{/if}}
+    <ul class="px-0" data-files>
+      {{#each this.files as |file|}}
+        <li class="flex items-center p-0">
```

Comment:
Yeah! 

I adjust the padding but I'm a little unclear how the Figma styles for the shadow outline translates to our tailwind classes.

Reply:
Ah! Up to you but I'd be happy to pair on this sometime this week!  Doesn't _have_ to be in this PR either so that this can be focused on the functionality and we can sweep through and make more nice-to-have downstream PRs later

Reply:
Moved this to: https://github.com/CrowdStrike/ember-toucan-core/issues/86